include/eve/module/core/regular/impl/simd/arm/sve/store.hpp

```diff
@@ -0,0 +1,47 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/category.hpp>
+#include <eve/detail/implementation.hpp>
+#include <eve/module/core/regular/unalign.hpp>
+
+namespace eve::detail
+{
+// Regular store
+template<real_scalar_value T, typename N, simd_compatible_ptr<wide<T, N>> Ptr>
+EVE_FORCEINLINE void
+store_(EVE_SUPPORTS(sve_), wide<T, N> v, Ptr p)
+requires(sve_abi<abi_t<T, N>> && !has_store_equivalent<wide<T, N>, Ptr>)
+{
+  auto ptr = unalign(p);
+  if constexpr( N() != expected_cardinal_v<T> ) memcpy(ptr, (T const*)(&v), N::value * sizeof(T));
```

Comment:
This looks bad. We should use a masked store. Just fwd to keep_first(N{}())

Reply:
Will fix